CenterForOpenScience/osf.io
# API should not return null log when the component inherits license from parent[OSF-7186]

## Purpose
Make the node api return node.license property instead of node_license field. node.license will return node.license if there is one or its parent's license.


## QA notes
1. create a project
2. create a component in the project
3. add a license to the project
4. view the component's node detail api
Before the change: 
The node_license should show up as "null" and there is no license field in realtionship field, while you can actually see a license on the component when visiting component overview page.
![screen shot 2016-11-02 at 10 45 25 am](https://cloud.githubusercontent.com/assets/4974056/19933376/80f5aecc-a0e9-11e6-93b0-8340c75df8b0.png)
![screen shot 2016-11-02 at 11 29 53 am](https://cloud.githubusercontent.com/assets/4974056/19935247/b6128dc2-a0ef-11e6-8b8e-74c22fcd1cd0.png)

After the change:
The node_license should show up as its parent project's node_license , and there is a license field in relationship which links to the same license as its parent.
![screen shot 2016-11-02 at 11 19 07 am](https://cloud.githubusercontent.com/assets/4974056/19934760/33a9290a-a0ee-11e6-8509-fe674fc95373.png)
![screen shot 2016-11-02 at 11 19 35 am](https://cloud.githubusercontent.com/assets/4974056/19934781/431baea8-a0ee-11e6-8873-f368c49c5e7e.png)






## Side effects




## Ticket


https://openscience.atlassian.net/browse/OSF-7186
